fronts-client/src/components/FrontsEdit/CollectionComponents/Collection.tsx

```diff
@@ -381,6 +386,17 @@ class Collection extends React.Component<CollectionProps, CollectionState> {
   private removeFrontCollection = () => {
     this.props.removeFrontCollection(this.props.frontId, this.props.id);
   };
+
+  private handleDeleteClick = () => {
+    this.setState({ isDeleteClicked: true });
+    const isConfirm = window.confirm(
```

Comment:
This simple approach is taken other places in codebase as well hence used the same.